src/shared/tomoyo-util.c

```diff
@@ -0,0 +1,41 @@
+/***
+  This file is part of systemd.
+
+  Copyright 2017 Shawn Landden
+
+  systemd is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as published by
+  the Free Software Foundation; either version 2.1 of the License, or
+  (at your option) any later version.
+
+  systemd is distributed in the hope that it will be useful, but
+  WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public License
+  along with systemd; If not, see <http://www.gnu.org/licenses/>.
+***/
+
+#include <unistd.h>
+
+#include "tomoyo-util.h"
+
+static const char version_path[] = "/sys/kernel/security/tomoyo/version";
```

Comment:
There is no need to define this as a variable. (It is static, so the compiler might optimize it away, but still.).
Just use `#define TOMOYO_VERSION_PATH ...` or even include it directly in the code, since it's only used in one place.